src/logic/ClientController.ts

```diff
@@ -577,10 +585,16 @@ export default class ClientController {
 						console.error(url + ' ' + getErrorMessage(e));
 					}
 				},
-				index * 1000
+				index * timeoutValue
 			);
 		});
 
+		window.setTimeout(() => {
+			if(failedLoadsToRetry.length > 0 && retryCount < 3){
+				this.onRoundContent(failedLoadsToRetry, retryCount++);
```

Comment:
I suppose, 0 will be always sent here as a second argument?

Reply:
yeah, fixed